modules/monitoring-managing-silences.adoc

```diff
@@ -14,3 +14,8 @@ Creating silences is useful in scenarios where you have received an initial aler
 When creating a silence, you must specify whether it becomes active immediately or at a later time. You must also set a duration period after which the silence expires.
 
 After you create silences, you can view, edit, and expire them.
+
+[NOTE]
+====
+When you create silences, they are replicated across Alertmanager pods, even after the pods are restarted. However, if you do not configure persistent storage for Alertmanager, it might not retain the silences after restarting the pods.
```

Comment:
```suggestion
When you create silences, they are replicated across Alertmanager pods. However, if you do not configure persistent storage for Alertmanager, silences might be lost if all Alertmanager pods restart at the same time for example.
```

Reply:
(it's better to add a `for example` or sth, as the silences could be lost in some other ways...)